trunner/config.py

```diff
@@ -12,6 +12,9 @@
 # Default time after pexpect will raise TIEMOUT exception if nothing matches an expected pattern
 PYEXPECT_TIMEOUT = 8
 
+# Available targets for test runner. By default test are run on every target except host-pc
+TARGETS = ['ia32-generic', 'host-pc']
```

Comment:
maybe we should have `ALL_TARGETS` and `DEFAULT_TARGETS` instead of some logic while parsing configs?